dataverse-webapp/src/main/webapp/dataverse_template.xhtml

```diff
@@ -51,6 +51,11 @@
     </h:head>
     <h:body styleClass="#{widgetWrapper.widgetView ? 'widget-view' : ''}">
         <script src="#{resource['js/dv_accessibility.js']}?version=#{systemConfig.getVersionWithBuild()}"></script>
+        <script src="#{resource['leaflet/leaflet.js']}"></script>
+        <script src="#{resource['js/drodb.js']}"></script>
+        <script>
+            window.Drodb = window.Drodb || initDrodb()
```

Comment:
I would not use name `Drodb` in the source code. Instead maybe we could name it in some way that it is consistent with naming that is more commonly used in this project. Maybe something like `DataverseJS` or something similar.